tests/messaging_tests/test_messaging.py

```diff
@@ -271,3 +271,44 @@ def test_failures_to_message_group(test_messenger):
     assert error_messages == [
         Message("Tab A", "Section A", ("A123", "A321", "A456"), "grouped message", "SomeInputFailure")
     ]
+
+
+@pytest.mark.parametrize(
+    "input_cell_indexes, expected_cell_indexes",
+    (
+        (("A1",), ("A1",)),
+        (("B1", "A1"), ("A1", "B1")),  # Sorts by column correctly
+        (("A2", "A1"), ("A1", "A2")),  # Sorts by row correctly
+        (("A2", "B1", "A1"), ("A1", "A2", "B1")),  # Sorts both correctly
+        (("A1", "B50", "B4", "A10"), ("A1", "A10", "B4", "B50")),  # Sorts row numbers numerically not lexicographically
```

Comment:
This test case doesn't prove numeric over lexicographic as expected result would be the same either way

Reply:
yep good spot, fixed 😓